### DISCUSSION TITLE
[工具] Java 下载主页

### DISCUSSION THREAD

**Discussion by zkitefly** [Meloong-Git/PCL#5270] [自定义主页]
[展示视频](https://www.123865.com/s/82uajv-rf2xH)

一个 Java 下载工具主页

数据来自 [foojayio/discoapi](https://github.com/foojayio/discoapi)

可以在 PCL 中选择你想要下载的 Java 下载！

目前可用！

主页地址：
```
https://vip.123pan.cn/1821946486/PCL2-java_download_page/Custom.xaml
```

https://github.com/zkitefly/PCL2-java_download_page

**Comment by LinQingYuu** [comment]
一个 403(123pan)
一个不加 Custom.xaml 404 (GitHub Page)
(

**Reply by zkitefly** [reply]
我也没说可以用了呀（

**Reply by LinQingYuu** [reply]
> 我也没说可以用了呀（

~（满试一试）~

**Comment by zkitefly** [comment]
@LTCatt cc ?

**Comment by CreeperIsASpy** [comment]
提点小意见：
主题页面大量内容堆叠大量卡片可以考虑使用`local:MyList`+`local:MyListItem`结合制作。
[Custom.txt](https://github.com/user-attachments/files/18319288/Custom.txt)

> 因为 github 不支持`xaml`，所以文件是`txt`格式，下载完了改一下拓展名就好啦!

**Comment by LTCatt** [comment]
![image](https://github.com/user-attachments/assets/1440cd87-5918-4e9e-b08b-423fd475e7f5)
首页里的这几个按钮完全没啥用，能在首页隐藏下么？

---

内部跳转所使用的 `pcl2-java-download-page.zkitefly.eu.org` 在国内部分区域存在访问问题。

**Reply by zkitefly** [reply]
可以，到时候解决（

---

到时候看看怎么解决（
